.circleci/config.yml

```diff
@@ -62,6 +62,33 @@ jobs:
       - run: docker login -u="${DOCKER_USERNAME}" -p="${DOCKER_PASSWORD}"
       - run: make docker-push
 
+  publish_release:
+    docker:
+      # Available from https://hub.docker.com/r/circleci/golang/
+      - image: circleci/golang:1.10
+    working_directory: /go/src/github.com/improbable-eng/thanos
+    steps:
+      - checkout
+      - setup_remote_docker:
+          version: 17.07.0-ce
+      - attach_workspace:
+          at: .
+      - run: make promu
+      - run: promu crossbuild tarballs
+      - run: promu checksum .tarballs
+      - run: promu release .tarballs
+      - store_artifacts:
+          path: .tarballs
+          destination: releases
```

Comment:
hm.. I assume this gives us `.tarballs` in some S3 bucket, thus available from CircleCI tab.

What about something using `ghr` to upload these to release page? https://medium.com/@jgautheron/publish-your-golang-binaries-in-github-with-circleci-e0b64cb21bf8 (This config is for CircleCI v1, but cannot find anything for v2)

Reply:
Could be interesting, we do it in Prometheus with promu, this is just copy-pasta from there.